instrumentation/netty/netty-4.1/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/Netty41ClientSslTest.java

```diff
@@ -0,0 +1,260 @@
+            satisfies(ExceptionAttributes.EXCEPTION_STACKTRACE, v -> v.isNotNull()),
+            equalTo(
+                ExceptionAttributes.EXCEPTION_TYPE,
+                thrownException.getCause().getClass().getCanonicalName()));
+
+    testing.waitAndAssertTraces(
+        trace ->
+            trace.hasSpansSatisfyingExactly(
+                span ->
+                    span.hasName("parent")
+                        .hasNoParent()
+                        .hasStatus(StatusData.error())
+                        .hasEventsSatisfyingExactly(
+                            event -> event.hasAttributesSatisfyingExactly(attributeAssertion)),
+                span ->
+                    span.hasName("RESOLVE")
+                        .hasKind(SpanKind.INTERNAL)
+                        .hasParent(trace.getSpan(0))
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(ServerAttributes.SERVER_ADDRESS, uri.getHost()),
+                            equalTo(ServerAttributes.SERVER_PORT, uri.getPort())),
+                span ->
+                    span.hasName("CONNECT")
+                        .hasKind(SpanKind.INTERNAL)
+                        .hasParent(trace.getSpan(0))
+                        .hasAttributesSatisfyingExactly(
+                            equalTo(NetworkAttributes.NETWORK_TRANSPORT, "tcp"),
+                            equalTo(NetworkAttributes.NETWORK_TYPE, "ipv4"),
+                            equalTo(ServerAttributes.SERVER_ADDRESS, uri.getHost()),
+                            equalTo(ServerAttributes.SERVER_PORT, uri.getPort()),
+                            equalTo(NetworkAttributes.NETWORK_PEER_PORT, uri.getPort()),
+                            equalTo(NetworkAttributes.NETWORK_PEER_ADDRESS, "127.0.0.1")),
+                span ->
+                    span.hasName("SSL handshake")
+                        .hasKind(SpanKind.INTERNAL)
+                        .hasParent(trace.getSpan(0))
+                        .hasStatus(StatusData.error())
+                        .hasEventsSatisfyingExactly(
+                            event ->
+                                event.hasAttributesSatisfyingExactly(
+                                    satisfies(
+                                        ExceptionAttributes.EXCEPTION_TYPE,
+                                        v ->
+                                            v.isEqualTo(
+                                                SSLHandshakeException.class.getCanonicalName())),
+                                    satisfies(
+                                        ExceptionAttributes.EXCEPTION_STACKTRACE,
+                                        v -> v.isInstanceOf(String.class)),
+                                    satisfies(
+                                        ExceptionAttributes.EXCEPTION_MESSAGE, v -> v.isNotNull())))
+                        .hasAttributesSatisfyingExactly(
```

Comment:
would `hasException(new SSLHandshakeException(null))` work?

Reply:
it works.